tests/ExampleApps/VolumeDemoApp/main.cpp

```diff
@@ -0,0 +1,173 @@
+// Include Radium base application and its simple Gui
+#include <Gui/BaseApplication.hpp>
+#include <Gui/RadiumWindow/SimpleWindowFactory.hpp>
+
+// include the Engine/entity/component interface
+#include <Core/Geometry/MeshPrimitives.hpp>
+#include <Engine/Scene/EntityManager.hpp>
+#include <Engine/Scene/GeometryComponent.hpp>
+#include <Engine/Scene/GeometrySystem.hpp>
+
+#include <QTimer>
+
+int main( int argc, char* argv[] ) {
+    //! [Creating the application]
+    Ra::Gui::BaseApplication app( argc, argv );
+    app.initialize( Ra::Gui::SimpleWindowFactory {} );
+    //! [Creating the application]
+
+#ifndef DEMO_FILE
```

Comment:
Is this define planned to be kept as this ?

Reply:
Yes. The symbol DEMO_FILE is defined in the CMakeLists.txt so that we are sure the path to the loaded file is correctly set. If this symbol is not define, the demo default to a simpler, synthetic scene.

Reply:
This is here because I asked if we could have the demo loading a file instead of displaying a sphere, it is more fancy.
But on the other hand we do not have a proper resource management for demos, so here comes the hack.